ChangeLog

```diff
@@ -1,4 +1,8 @@
 
+2022-05-25  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
+
+	* NEWS: mention new -std options for GCOS 7 dialect
```

Comment:
nah, can be dropped (but better consider it first)

Reply:
So it's only mentioned in NEWS, right?

Reply:
yes (and cobc/Changelog as you would put that in cobc/help.c, too ;-)